matrix_sdk_base/src/store/mod.rs

```diff
@@ -196,6 +196,20 @@ pub trait StateStore: AsyncTraitDeps {
         &self,
         event_type: EventType,
     ) -> Result<Option<Raw<AnyBasicEvent>>>;
+
+    /// Get an event out of the room account data store.
+    ///
+    /// # Arguments
+    ///
+    /// * `room_id` - The id of the room for which the room account data event should
+    /// be fetched.
+    ///
+    /// * `event_type` - The event type of the room account data event.
```

Comment:
There are two lists with one item each here. It should probably be one list.

```suggestion
    /// * `room_id` - The id of the room for which the room account data event should
    /// be fetched.
    /// * `event_type` - The event type of the room account data event.
```

Reply:
I would agree but that's how everything is documented.

Reply:
Huh, weird.

Reply:
Don't tell me I've been doing lists the wrong way the whole time :sweat_smile:

Reply:
I thought it was a voluntary choice, somehow 🙂

Reply:
Well they render fine in the docs and I prefer having a bit of whitespace since vim + LSP renders this in a hover window as is without any whitespace.

Reply:
rustdoc seems to render it with one `ul` element too, I would have expected two. Well I guess then it doesn't matter.

Reply:
Let's leave the whitespace then, it is useful for vim/emacs users.